src/styles/blocks/header.scss

```diff
@@ -0,0 +1,15 @@
+.header {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 50px;
+  }
```

Comment:
Same here, do you really need additional container? Let's apply this styles to `header`